### PR TITLE
[KYUUBI #684]Improve kyuubi ctl verbose output

### PR DESCRIPTION
**Pull request by hddong**
### _Why are the changes needed?_

Improve kyuubi ctl verbose output.

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.readthedocs.io/en/latest/tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

**Comment by yaooqinn** [comment]
IMO, verbose is for output? not related to logging?

**Comment by hddong** [comment]
@yaooqinn: Sorry for feedback late. It seems that slf4j not support custom level, temporarily use log4j instead.
Result show:
```
2021-06-16 14:59:18.363 INFO kyuubi.Utils: Loading Kyuubi properties from /root/hongdd/kyuubi-1.3.0-SNAPSHOT-bin-spark-3.0-hadoop3.2/conf/kyuubi-defaults.conf
2021-06-16 14:59:18.579 WARN util.NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
2021-06-16 14:59:18.654 VERBOSE ctl.ServiceControlCli$$anon$1$$anon$2: Zookeeper quorum is not specified, use value from default conf:k8s2:2181,k8s3:2181,k8s4:2181
2021-06-16 14:59:18.655 VERBOSE ctl.ServiceControlCli$$anon$1$$anon$2: Zookeeper namespace is not specified, use value from default conf:kyuubi
2021-06-16 14:59:18.659 VERBOSE ctl.ServiceControlCli$$anon$1$$anon$2: version is not specified, use built-in KYUUBI_VERSION:1.3.0-SNAPSHOT
2021-06-16 14:59:18.666 VERBOSE ctl.ServiceControlCli$$anon$1: Parsed arguments:
  action                  LIST
  service                 SERVER
  zkQuorum                k8s2:2181,k8s3:2181,k8s4:2181
  namespace               kyuubi
  user                    null
  host                    null
  port                    null
  version                 1.3.0-SNAPSHOT
  verbose                 true
    
2021-06-16 14:59:18.730 INFO imps.CuratorFrameworkImpl: Starting
2021-06-16 14:59:18.752 INFO zookeeper.ClientCnxn: Opening socket connection to server k8s4/10.142.19.49:2181. Will not attempt to authenticate using SASL (unknown error)
2021-06-16 14:59:18.758 INFO zookeeper.ClientCnxn: Socket connection established to k8s4/10.142.19.49:2181, initiating session
```

**Comment by yaooqinn** [comment]
```
VERBOSE ctl.ServiceControlCli$$anon$1$$anon$2 

```

looks strange

**Comment by yaooqinn** [comment]
`if (!verbose)` then we set our log level to ERROR?

for default mode, we can run  `kyuubi-ctl` silently like `kubectl` without any log.

for example,
```
kubectl get pods -nspark
NAME            READY   STATUS              RESTARTS   AGE
kyuubi-exec-1   1/1     Running             0          69s
kyuubi-exec-2   1/1     Running             0          69s
kyuubi-exec-4   1/1     Running             0          69s
kyuubi-exec-5   1/1     Running             0          69s
kyuubi-exec-6   0/1     ContainerCreating   0          61s
```

**Comment by hddong** [comment]
> `if (!verbose)` then we set our log level to ERROR?
> 
> for default mode, we can run `kyuubi-ctl` silently like `kubectl` without any log.
> 
> for example,
> 
> ```
> kubectl get pods -nspark
> NAME            READY   STATUS              RESTARTS   AGE
> kyuubi-exec-1   1/1     Running             0          69s
> kyuubi-exec-2   1/1     Running             0          69s
> kyuubi-exec-4   1/1     Running             0          69s
> kyuubi-exec-5   1/1     Running             0          69s
> kyuubi-exec-6   0/1     ContainerCreating   0          61s
> ```

**Comment by hddong** [comment]
Consensus, Had clean up the verbose information fro kyuubi-ctl. 
```
# bin/kyuubi-ctl list

           Zookeeper service nodes           
+----------+----------+----------+--------------+
|Namespace |   Host   |   Port   |   Version    |
+----------+----------+----------+--------------+
| /kyuubi  | 0.0.0.0  |  10009   |1.3.0-SNAPSHOT|
+----------+----------+----------+--------------+
1 row(s)
```
Other, I wonder if we should change default zookeeper level to WARN, zk logs are too verbose.

**Comment by yaooqinn** [comment]
thank you！@hddong, seems we do not add a new VERBOSE log level?

**Comment by yaooqinn** [comment]
> Consensus, Had clean up the verbose information fro kyuubi-ctl.
> 
> ```
> # bin/kyuubi-ctl list
> 
>            Zookeeper service nodes           
> +----------+----------+----------+--------------+
> |Namespace |   Host   |   Port   |   Version    |
> +----------+----------+----------+--------------+
> | /kyuubi  | 0.0.0.0  |  10009   |1.3.0-SNAPSHOT|
> +----------+----------+----------+--------------+
> 1 row(s)
> ```
> 
> Other, I wonder if we should change default zookeeper level to WARN, zk logs are too verbose.

yes we can have another PR for this

File: kyuubi-common/src/main/scala/org/apache/kyuubi/logger/KyuubiLevel.scala [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+import org.apache.log4j.{Level, Priority}
+
+/**
+ * Kyuubi custom log level.
+ */
+object KyuubiLevel {
+  final val VERBOSE_INT: Int = 19999
+
+  final val VERBOSE = new KyuubiLevel(VERBOSE_INT, "VERBOSE", 6)
```
Comment (hddong) [inline kyuubi-common/src/main/scala/org/apache/kyuubi/logger/KyuubiLevel.scala:28]: @yaooqinn: Had add `VERBOSE` level here, lower than `INFO`.
use `--verbose` or set `log4j.logger.org.apache.kyuubi.ctl.ServiceControlCli=VERBOSE#org.apache.kyuubi.logger.KyuubiLevel`
Show example:
```
2021-06-18 14:03:12.160 WARN util.NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
2021-06-18 14:03:12.233 VERBOSE ctl.ServiceControlCli: Zookeeper quorum is not specified, use value from default conf:k8s2:2181,k8s3:2181,k8s4:2181
2021-06-18 14:03:12.235 VERBOSE ctl.ServiceControlCli: Zookeeper namespace is not specified, use value from default conf:kyuubi
2021-06-18 14:03:12.238 VERBOSE ctl.ServiceControlCli: version is not specified, use built-in KYUUBI_VERSION:1.3.0-SNAPSHOT
2021-06-18 14:03:12.247 VERBOSE ctl.ServiceControlCli: Parsed arguments:
  action                  LIST
  service                 SERVER
  zkQuorum                k8s2:2181,k8s3:2181,k8s4:2181
```

**Comment by hddong** [comment]
@yaooqinn: please have a review again when free.
The result shows like:
```
# bin/kyuubi-ctl list
 /kyuubi   0.0.0.0    10009   1.3.0-SNAPSHOT

# bin/kyuubi-ctl list --verbose

           Zookeeper service nodes           
+----------+----------+----------+--------------+
|Namespace |   Host   |   Port   |   Version    |
+----------+----------+----------+--------------+
| /kyuubi  | 0.0.0.0  |  10009   |1.3.0-SNAPSHOT|
+----------+----------+----------+--------------+
1 row(s)

```

File: kyuubi-ctl/src/main/scala/org/apache/kyuubi/ctl/CommandLineUtils.scala [hunk 2]
```diff
@@ -74,8 +74,9 @@ private[kyuubi] object Tabulator {
     if (str == null) 0 else str.length + fullWidthRegex.findAllIn(str).size
   }
 
-  def format(title: String, header: Seq[String], rows: Seq[Seq[String]]): String = {
-    val data = Seq(header).union(rows)
+  def format(title: String, header: Seq[String], rows: Seq[Seq[String]],
+             verbose: Boolean): String = {
```
Comment (yaooqinn) [inline kyuubi-ctl/src/main/scala/org/apache/kyuubi/ctl/CommandLineUtils.scala:78]: indention

File: kyuubi-ctl/src/test/scala/org/apache/kyuubi/ctl/ServiceControlCliSuite.scala [hunk 3]
```diff
@@ -120,9 +120,14 @@ class ServiceControlCliSuite extends KyuubiFunSuite with TestPrematureExit {
   }
 
   /** Get the rendered service node info without title */
-  private def getRenderedNodesInfoWithoutTitle(nodesInfo: Seq[ServiceNodeInfo]): String = {
-    val renderedInfo = renderServiceNodesInfo("", nodesInfo)
-    renderedInfo.substring(renderedInfo.indexOf("|"))
+  private def getRenderedNodesInfoWithoutTitle(nodesInfo: Seq[ServiceNodeInfo],
+                                               verbose: Boolean): String = {
```
Comment (yaooqinn) [inline kyuubi-ctl/src/test/scala/org/apache/kyuubi/ctl/ServiceControlCliSuite.scala:124]: indention

**Review by yaooqinn — APPROVED** [review]

**Comment by hddong** [comment]
@yaooqinn: Thanks for your review, I will fix these indention in next PR.

**Comment by yaooqinn** [comment]
thanks~

**Pull request merged**